LondonBritishLibrary/orient/BLorient12996.xml

```diff
@@ -0,0 +1,292 @@
+                                    Magistro fallit</foreign>.</note>
+                                <textLang xml:lang="la"/>
+                                <msItem xml:id="p1_i1.1">
+                                    <locus from="1r"/>
+                                    <title ref="LIT000000000000000000000#Script">Script and Phonetics</title>
+                                    <note>Introduction to the Ethiopic alphasyllabary and the phonetics of the language. The author gives the Arabic letters
+                                        corresponding to the Amharic characters and also indicates the pronunciation with reference to German (indicated as 
+                                        <foreign xml:lang="la">Germ.</foreign>), Czech (indicated as <foreign xml:lang="la">Boem.</foreign>, Italian 
+                                        (indicated as <foreign xml:lang="la">Ital.</foreign>, and French <foreign xml:lang="la">Gall.</foreign>. There 
+                                        are also references to Persian. The author notes the letters <foreign xml:lang="am">ፙ</foreign>, transliterated as 
+                                        'moa', a form similar to <foreign xml:lang="am">ፙ</foreign>, but with a mark of the second order on the right side 
+                                        and transliterated as 'mha' (sic) as well as <foreign xml:lang="am">ኸ</foreign>, transliterated as 'chha', etc.
+                                        <foreign xml:lang="am">ጸ</foreign> and <foreign xml:lang="am">ፀ</foreign> are both transliterated as 'za',
+                                        <foreign xml:lang="am">ጀ</foreign> is transliterated as 'gia'. The series of ž is noted by 
+                                        <foreign xml:lang="am">ዠ</foreign>etc. (a character with a mark on the lower left side, which signifies the sixth 
+                                        order).
+                                        The pronunciation of <foreign xml:lang="am">ቸ</foreign> is indicated as 'cia' (but on <locus target="#5r"/>:
+                                        <foreign xml:lang="cs">cža</foreign> <foreign xml:lang="la">Boem.</foreign>) and that of
+                                        <foreign xml:lang="am">ጨ</foreign> as 'cca' (but  <foreign xml:lang="cs">cža</foreign> (sic!) 
+                                        <foreign xml:lang="la">Boemice</foreign>). The author notes the difference between the first and the fourth orders
+                                        only as a difference in length but indicates the two functions of the sixth.</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.2">
+                                    <locus target="#6r"/>
+                                    <title ref="LIT00000000000000000000000#Oration1">De partibus Orationis</title>
+                                    <note>Some orations as language samples.</note>
```

Comment:
Does the Latin not mean "On the parts of speech"?

Reply:
Yes, but you also provide your own English translation in a note, as I understand it.